src/functionalTest/java/uk/gov/hmcts/divorce/systemupdate/event/SystemNotifyApplicantDisputeFormOverdueFT.java

```diff
@@ -67,8 +68,10 @@ public void shouldSearchForDisputeFormOverdueCases() {
         searchForCasesWithQuery(query)
             .forEach(caseDetails -> {
                 assertThat(caseDetails.getState().equals(Holding));
-                assertThat(caseDetails.getData().get(NOTIFICATION_SENT_FLAG)).isNotEqualTo(YesOrNo.YES);
-                assertThat(DISPUTE_DIVORCE.getType().equals(caseDetails.getData().get("howToRespondApplication")));
+                CaseData caseData = getCaseData(caseDetails.getData());
+                assertThat(DISPUTE_DIVORCE.getType().equals(caseData.getAcknowledgementOfService().getHowToRespondApplication()));
+                assertThat(!caseData.getApplication().getIssueDate().plusDays(10).isAfter(LocalDate.now()));
```

Comment:
you could use before rather than ! at the beginning

Reply:
but it has to be on or before. when I used before the one on the same day failed

Reply:
can use `isBeforeOrEqualTo`